### PR TITLE
Renamed mygardens to myniwa

### PR DESCRIPTION
**Pull request by curtisahuang**
## Description

Renaming

## Closes issue(s)

Renamed mygardnes -> myniwa pages

## Screenshots

If you have image share...

## Changes include

- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [X] I have tested this code

### CODE REVIEW

**Comment by daiHash** [comment]
Please remember to assign yourself and add status label for PR

**Comment by curtisahuang** [comment]
> 
> 
> Please remember to assign yourself and add status label for PR.
> Is this PR WIP or good for review?

Good for review.

File: src/pages/MyNiwa/index.tsx [hunk 1]
```diff
@@ -38,12 +38,12 @@ const useStyles = makeStyles((theme: Theme) =>
...
 );
 
-export const MyGardens = () => {
+export const MyNiwa = () => {
   const classes = useStyles();
   const { userData } = useUserState();
   const [gardensApi, getUserGardens] = useApi(getGardens);
 
-  const myGardens = useMemo(() => gardensApi.response ?? [], [gardensApi]);
+  const myniwa = useMemo(() => gardensApi.response ?? [], [gardensApi]);
```
Comment (daiHash) [inline src/pages/MyNiwa/index.tsx:47]: I think for the state is ok to keep using garden terminology maybe `gardens` would be good

**Review by daiHash — APPROVED** [review]
Nice LGTM!

**Review by conalli — APPROVED** [review]
lgtm

**Pull request merged**